codespell_lib/data/dictionary.txt

```diff
@@ -567,9 +569,10 @@ acquited->acquitted
 acqure->acquire
 acqured->acquired
 acqures->acquires
+acquries->acquires
```

Comment:
This should perhaps suggest equerries too.

Reply:
added